website/docs/r/oracledatabase_exadata_infrastructure.html.markdown

```diff
@@ -0,0 +1,106 @@
+---
+subcategory: "App Service"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_oracledatabase_exadata_infrastructure"
+description: |-
+  Manages a Exadata Infrastructure.
+---
+
+# azurerm_oracledatabase_exadata_infrastructure
+
+Manages a Exadata Infrastructure.
+
+## Example Usage
+
+```hcl
+resource "azurerm_resource_group" "example" {
+  name     = "example-resources"
+  location = "West Europe"
+}
+
+resource "azurerm_oracledatabase_exadata_infrastructure" "example" {
+  name = "example-exadata-infra"
+  resource_group_name = azurerm_resource_group.example.name
+  location = azurerm_resource_group.example.location
+  zones = [ "1" ]
+  display_name = "example-exadata-infra"
+  storage_count = 3
+  compute_count = 2
+  shape = "Exadata.X9M"
+}
+```
+
+## Arguments Reference
+
+The following arguments are supported:
+
+* `compute_count` - (Required) The number of compute servers for the cloud Exadata infrastructure.
```

Comment:
might i suggest
```suggestion
* `compute_server_count` - (Required) The number of compute servers for the cloud Exadata infrastructure.
```

Reply:
I agree that this is clearer - however we have a requirement to try and ensure the argument names are the same as those in the official OCI Terraform provider:
https://registry.terraform.io/providers/oracle/oci/latest/docs/resources/database_cloud_exadata_infrastructure#compute_count